src/node.js

```diff
@@ -81,10 +82,11 @@ function prepareIPFSConfig () {
 }
 
 async function start () {
+  const logger = new Logger("Pinning Server")
```

Comment:
Defining a logger at the app's entry point and using it throughout would mean we are using the same logger everywhere (which means all log entries would have the same name), whereas it's more helpful to have a logger for each component/module.

If we continue down the path of a top-level logger passed into other modules, we could solve this by creating child loggers on each of the modules we pass it into, so that they (and the entries they generate) can be tied to that module.

However, the pattern I find easier to work with is to just require bunyan at the top of each module and create a logger for that module. It's not hierarchical, but we avoid adding the complexity of DI for this.

If we do that, however, we either have to set the additional global options (such a as `version` field) at the top of each module, or create out own logger module (like you've done here, but not as a class, but a simple wrapper for bunyan's `createLogger` function to maintain consistency)